packages/ui/src/components/Table/Table.tsx

```diff
@@ -22,9 +22,16 @@ type Props = {
 const TableComponent: React.FC<Props> = ({ children, loading, error }) => {
 	return (
 		<Box className={styles.container}>
-			{loading && 'Loading...'}
-			{!!error && error}
-			{!loading && !error && children}
+			{Children.map(children, (child: any) => {
```

Comment:
I think we are going to get this warning on all subsequent PRs. Perhaps this is another reason to have people explicitly render `Table.Loading` and `Table.Error` instead of trying to support the `loading` and `error` props on the top-level 🤔